packages/cornucopia/src/stores/localstorage.ts

```diff
@@ -26,21 +21,30 @@ export class CashuLocalStorage implements CashuStorage<StartProofSelection> {
     localStorage.setItem(this.PROOFS_KEY, JSON.stringify(proofs));
   }
 
+  private async selectProofs(filter: ProofFilter) {
+    let proofs = this.loadAllProofs();
+
+    const unlocked = await this.locker.getUnlocked(proofs);
+    if (filter.locked) proofs = proofs.filter((p) => !unlocked.includes(p));
+    else proofs = unlocked;
+
+    if (filter.expired) proofs = await this.locker.getExpired(proofs);
+
+    if (filter.amount !== undefined) {
+      const { send } = selectProofsToSend(proofs, filter.amount);
+      proofs = send;
```

Comment:
Same here as my comment on the sqlite store. I think that this is the correct order. I think the filter should be described something like: amount AND (locked OR expired)